src/main/services/pre-api/pre-client.ts

```diff
@@ -25,7 +31,55 @@ export class PreClient {
 
       return response.data as Recording;
     } catch (e) {
-      return null;
+      if (e.response.status === 404) {
+        return null;
+      }
+
+      this.logger.error(e);
+      throw e;
+    }
+  }
+
+  public async getRecordingPlaybackData(id: string): Promise<RecordingPlaybackData | null> {
+    const url = config.get('ams.flowUrl') as string;
+    const key = config.get('ams.flowKey') as string;
+    const axiosClient = axios.create(); // TODO: move AMS playback logic to API and use instead of flow above
+
+    delete axiosClient.defaults.headers.common['Ocp-Apim-Subscription-Key'];
+    delete axiosClient.defaults.headers.common['X-User-Id'];
+
+    try {
+      const response = await axiosClient.post(
+        url,
+        {
+          RecordingId: id,
+          AccountID: 'e1b7674b-b94d-4e07-9957-48345845885a', // TODO: get authenticated user id. Remove when AMS playback logic is moved to API
+        },
+        {
+          headers: {
+            'Content-Type': 'application/json',
```

Comment:
```suggestion
```
I think this is set in app.ts already? `axios.defaults.headers.post['Content-Type'] = 'application/json';`